sklearn/linear_model/tests/test_least_angle.py

```diff
@@ -360,6 +360,7 @@ def test_lars_n_nonzero_coefs(verbose=False):
     assert_equal(len(lars.alphas_), 7)
 
 
+@ignore_warnings
```

Comment:
There is a ConvergenceWarning caused due to the the very low value of tolerance set in L336, It can be fixed by setting tol to 1e-4